sprout/appliances/views.py

```diff
@@ -391,7 +391,12 @@ def my_appliances(request, show_user="my"):
         end_index -= start_index
         start_index = 0
     pages = pages[start_index:end_index]
-    groups = Group.objects.order_by("id")
+    available_groups = Group.objects.filter(
```

Comment:
I was trying to find out how to do it the best way "Gimme all streams that have at least one template" but I haven't figured out how to do it in the ORM ...